samples/cpp/image_generation/benchmark_image.cpp

```diff
@@ -0,0 +1,266 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/genai/image_generation/text2image_pipeline.hpp"
+#include "openvino/genai/image_generation/image2image_pipeline.hpp"
+#include "openvino/genai/image_generation/inpainting_pipeline.hpp"
+#include <cxxopts.hpp>
+#include <chrono>
+#include "imwrite.hpp"
+#include "load_image.hpp"
+#include "progress_bar.hpp"
+
+inline void print_one_generate(ov::genai::ImageGenerationPerfMetrics& metrics, std::string prefix, int idx) {
+    std::string prefix_idx = "[" + prefix + "-" + std::to_string(idx) + "]";
+    std::cout << "\n";
+    std::cout << prefix_idx << " generate time: " << metrics.get_generate_duration()
+              << " ms, total infer time:" << metrics.get_all_infer_duration()
+              << " ms" << std::endl;
+    std::cout << prefix_idx << " encoder infer time: " << metrics.get_encoder_infer_duration() << " ms"<< std::endl;
```

Comment:
Is encoder infer time assumes both text encoder and image encoders?

for image to image and inpainting we also run VAE encoder to convert initial image to latent space. I think we need different values for text encoder (runs in all types of pipelines) and image encoder (only when initial image is passed)